### PR TITLE
chore(babel): upgrade babel to 6

### PR DESCRIPTION
**Pull request by underbyte**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 language: node_js
 
 node_js:
-  - "0.10"
+  - "5.3.0"
```
Comment (underbyte) [inline .travis.yml]: We should at least standardize on version 5.0 of node. I got the babel error below only on node 0.10.

```
Running "browserify:test" (browserify) task
>> Error: module "babel-runtime/core-js/object/get-prototype-of" not found from "/Users/canon-react/transpiled/Criteria.js"
Warning: Error running grunt-browserify. Use --force to continue.
```

**Comment by adampickeral** [comment]
have you tested the minified file to make sure it works with reach yet?

File: .travis.yml (hunk 1 above)
Comment (adampickeral) [inline .travis.yml]: you probably got that because of the different npm version that comes with node v5.

**Comment by adampickeral** [comment]
squash commits and fix commit message

File: src/index.js [hunk 2]
```diff
@@ -16,7 +16,7 @@ import ProgressBar from './ProgressBar';
 import StatusIndicator from './StatusIndicator';
 import TooltipTrigger from './TooltipTrigger';
 
-export default {
+window.canonReact = {
```
Comment (adampickeral) [inline src/index.js]: what if instead of `export default { }` you just did `export { }`

Comment (adampickeral) [inline src/index.js]:
```
export { Button, ButtonGroup, Criteria... }
```

File: package.json [hunk 3]
```diff
@@ -39,23 +39,25 @@
...
   },
   "peerDependencies": {
     "react": ">=0.13.3"
   },
   "devDependencies": {
-    "babel": "^5.5.8",
-    "babel-eslint": "^4.1.1",
-    "babel-runtime": "^5.5.8",
-    "babelify": "^6.1.2",
+    "babel": "^6.3.26",
```
Comment (adampickeral) [inline package.json]: use exact versions for all the dependencies you've added/updated please

**Pull request merged**